styles/components/_mobile_menu.scss

```diff
@@ -0,0 +1,51 @@
+.MobileMenu {
+
+  &__container {
+    transform: translateY(-200%);
+    transition: transform 0.35s ease-out;
+  }
+
+  &__mobile-banner {
+    @extend .box-shadow;
+
+    &__banner-icon {
```

Comment:
Same here `MobileMenu__mobile-banner__banner-icon` is crazy. Let's just call this: `MobileMenu__banner-icon`